### PR TITLE
yarn tsc works without special script

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,14 +63,14 @@
...
-    "test:debug-examples": "npm run tsc && npm run schema && mocha --recursive --debug-brk --inspect build/examples",
+    "test:debug": "tsc && mocha --recursive --debug-brk --inspect build/test",
+    "test:debug-examples": "tsc && npm run schema && mocha --recursive --debug-brk --inspect build/examples",
     "mocha:test": "mocha --require source-map-support/register --reporter dot --recursive build/test",
     "mocha:examples": "mocha --require source-map-support/register --reporter dot --recursive build/examples",
 
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
     "watch:build": "watchify src/index.ts -p tsify -v -d -s vl -o 'exorcist build/vega-lite.js.map > build/vega-lite.js'",
-    "watch:tsc": "npm run tsc -- -w",
+    "watch:tsc": "tsc -- -w",
```
Comment (kanitw) [inline package.json]: do you still need `--` here?

Comment (domoritz) [reply package.json]: Nope. It's not needed anymore.

Comment (domoritz) [reply package.json]: Should I just delete this entry then? You can just call `yarn tsc -w`.

Comment (kanitw) [reply package.json]: I never use it so fine with me.

**Pull request merged**